worlds/hylics2/__init__.py

```diff
@@ -149,8 +134,7 @@ def fill_slot_data(self) -> Dict[str, Any]:
         slot_data: Dict[str, Any] = {
             "party_shuffle": self.options.party_shuffle.value,
             "medallion_shuffle": self.options.medallion_shuffle.value,
-            "random_start" : self.options.random_start.value,
```

Comment:
I would leave this line in, and change it to 
`"random_start": int(self.options.start_location != "waynehouse")`
since then it would still work with the current client.

Reply:
I changed this in the client with the PR I have linked, since this would make having that information completely useless except for preserving the identical client.

Reply:
I know, but personally I still prefer the option of preserving the current client, first because Hylics 2 doesn't have any kind of mod installer/manager to make things easier, and second because I also think it would be confusing to have a newest release for an indefinite amount of time that doesn't work with the latest live version of Archipelago, until RCs are over and a new version is released.

Reply:
Alright, I don't mind doing that and closing the other PR

Reply:
Done.